src/routes/index.js

```diff
@@ -9,14 +9,33 @@ const user = require('../mocks/user')
 const tabelaFarmaceutica = require('../mocks/tabelaFarmaceutica')
 const dadosFarmacia = require('../mocks/farmacia')
 
+const createPrescricao = require('./prescricao/create')
+const destroyPrescricao = require('./prescricao/destroy')
+const editPrescricao = require('./prescricao/edit')
+const getPrescricao = require('./prescricao/get')
+const updatePrescricao = require('./prescricao/update')
 
+const editAcolhido = require('./acolhido/edit')
+const getAcolhido = require('./acolhido/get')
+const updateAcolhido = require('./acolhido/update')
+const createAcolhido = require('./acolhido/create')
 const allRoutes = models => ({
   listaAcolhidos: listaAcolhidos(models.Acolhido),
   home: home(),
   about: about(user),
-  acolhido: acolhido(models.Acolhido),
+  // acolhido: acolhido(models.Acolhido),
```

Comment:
Mais comentários 
![judging you](https://media0.giphy.com/media/mxrwSmquLokMw/giphy-downsized.gif)